### PR TITLE
[Chore]Remove ConnectorV1 from the dev branch

### PR DESCRIPTION
**Pull request by CalvinKirs**
V1 has not been updated for a long time, and it has also tended to be stable. 
As a development branch, we are currently focusing on V2-related development, so I will remove it from `dev` branch.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -82,7 +82,6 @@
         <module>seatunnel-core</module>
         <module>seatunnel-transforms</module>
```
Comment (EricJoy2048) [inline pom.xml:83]: delete `seatunnel-transforms` ?

Comment (CalvinKirs) [reply pom.xml:83]: https://github.com/apache/incubator-seatunnel/pull/3396
https://github.com/apache/incubator-seatunnel/pull/3409
we still have code dependencies, we can do this later

**Review by EricJoy2048 — DISMISSED** [review]

**Comment by hailin0** [comment]
Please check update the following file：

https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/src/main/assembly/assembly-bin-ci.xml#L144
https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/src/main/assembly/assembly-bin.xml#L49

**Comment by hailin0** [comment]
Please check update the following files：

`bin/install-plugin.sh`
`plugin-mapping.properties`

**Comment by CalvinKirs** [comment]
> Please check update the following files：
> 
> `bin/install-plugin.sh` `plugin-mapping.properties`

Thx, update  `plugin-mapping.properties`, As for `bin/install-plugin.sh`, I suggest we keep it, if some user wants to choose to mix, he can still do so, copy `start (v1)` and `connector` jar to `lib`, WDYT?

**Comment by hailin0** [comment]
> > Please check update the following files：
> > `bin/install-plugin.sh` `plugin-mapping.properties`
> 
> Thx, update `plugin-mapping.properties`, As for `bin/install-plugin.sh`, I suggest we keep it, if some user wants to choose to mix, he can still do so, copy `start (v1)` and `connector` jar to `lib`, WDYT?


Please revert plugin-mapping.properties update, if you consider mixed use

**Comment by CalvinKirs** [comment]
> > > Please check update the following files：
> > > `bin/install-plugin.sh` `plugin-mapping.properties`
> > 
> > 
> > Thx, update `plugin-mapping.properties`, As for `bin/install-plugin.sh`, I suggest we keep it, if some user wants to choose to mix, he can still do so, copy `start (v1)` and `connector` jar to `lib`, WDYT?
> 
> Please revert plugin-mapping.properties update, if you consider mixed use

revert. But it doesn't really matter. I don't recommend maintaining this in dev. If necessary, a better way is to add it yourself. After all, we no longer intend to maintain this, we just provide a hybrid method.

**Comment by hailin0** [comment]
Please check update  `.github/workflows/*.yml`

**Review by hailin0 — APPROVED** [review]

**Review by dijiekstra — APPROVED** [review]
LGTM

**Pull request merged**